client.lua

```diff
@@ -29,6 +29,13 @@ local function SplitStr(inputstr, sep)
 end
 
 local function connecttoradio(channel)
+    if channel > Config.MaxFrequency or channel <= 0 then QBCore.Functions.Notify(Lang:t('restricted_channel_error'), 'error') return false end
+    if Config.RestrictedChannels[channel] ~= nil then
+        if not Config.RestrictedChannels[channel][PlayerData.job.name] and PlayerData.job.onduty then
```

Comment:
Thank you for the changes. I think those conditions can be simplified though, more closer to the original code by just changing the comparison from `and` to `or`, and flipping the second condition.

I believe this would suffice and offer the same functionality:
```lua
        if not Config.RestrictedChannels[channel][PlayerData.job.name] or not PlayerData.job.onduty then
```
It reads as, if job is not whitelisted for the channel, it's a true fail condition, `or` if not on duty, it's a true fail condition.
This would mean if their job is not on the whitelist, it won't check the second comparison, and if their job is on the whitelist, it'll check the second comparison for whether they're on duty.